src/components/LoginBox/LoginBox.js

```diff
@@ -54,7 +69,7 @@ class LoginBox extends PureComponent {
                   type="text"
                 />
                 <div className="errors">
-                  { errors.email && errors.email }
```

Comment:
This will always display `errors.email`. If you look at both possibilities:
1) errors.email is falsey, && not passed -> errors.email (the first) displayed
2) errors.email is truthy, && passed -> errors.email (the second) displayed

Reply:
I thought && in a scenario like this was the equivalent of a x ? y : z without the else, so is errors.email truthy or falsey?, if falsey, do nothing, else errors.email.